diff --git a/lib/active_support.rb b/lib/active_support.rb
index 7ebb3c4..63c76cc 100644
--- a/lib/active_support.rb
+++ b/lib/active_support.rb
@@ -51,7 +51,7 @@ module ActiveSupport
   autoload :XmlMini, 'active_support/xml_mini'
 end
 
-require 'active_support/vendor'
+require 'i18n'
 require 'active_support/core_ext'
 require 'active_support/dependencies'
 require 'active_support/json'
diff --git a/lib/active_support/json/encoding.rb b/lib/active_support/json/encoding.rb
index ffd2050..7b30ee8 100644
--- a/lib/active_support/json/encoding.rb
+++ b/lib/active_support/json/encoding.rb
@@ -81,10 +81,7 @@ module ActiveSupport
 end
 
 # Hack to load json gem first so we can overwrite its to_json.
-begin
   require 'json'
-rescue LoadError
-end
 
 require 'active_support/json/variable'
 require 'active_support/json/encoders/date'
diff --git a/lib/active_support/test_case.rb b/lib/active_support/test_case.rb
index bde74fa..8421241 100644
--- a/lib/active_support/test_case.rb
+++ b/lib/active_support/test_case.rb
@@ -5,7 +5,6 @@ require 'active_support/testing/deprecation'
 require 'active_support/testing/declarative'
 
 begin
-  gem 'mocha', ">= 0.9.7"
   require 'mocha'
 rescue LoadError
   # Fake Mocha::ExpectationError so we can rescue it in #run. Bleh.
diff --git a/lib/active_support/testing/performance.rb b/lib/active_support/testing/performance.rb
index f8d12e8..38837a3 100644
--- a/lib/active_support/testing/performance.rb
+++ b/lib/active_support/testing/performance.rb
@@ -1,9 +1,6 @@
-require 'rubygems'
-gem 'ruby-prof', '>= 0.6.1'
 require 'ruby-prof'
 
 require 'fileutils'
-require 'rails/version'
 
 module ActiveSupport
   module Testing
@@ -124,6 +121,7 @@ module ActiveSupport
 
         def environment
           unless defined? @env
+            require 'rails/version'
             app = "#{$1}.#{$2}" if File.directory?('.git') && `git branch -v` =~ /^\* (\S+)\s+(\S+)/
 
             rails = Rails::VERSION::STRING
